src/richie/apps/courses/models/course.py

```diff
@@ -346,16 +346,10 @@ class CoursePluginModel(PagePluginMixin, CMSPlugin):
         on_delete=models.CASCADE,
         related_name="course_plugins",
         limit_choices_to={
-            # There's a draft and a public course attached to the draft and
-            # the public parent. Without the first condition, 4 options are
-            # availables for each course. The second condition makes sure the
-            # parent is not a course. The third option filters out public
-            # course. The fourth option makes sure only courses show up.
-            # If there's is no parent, the course is filtered out.
```

Comment:
don't remove this explanation. I had to re-read it in order to understand what we were doing here.
The small comments below can be additional.

You can improve it by replacing:
```
If there's is no parent, the course is filtered out.
```
by:
```
If there's is no published parent, the course is filtered out.
```